docs/components/datetimepicker.md

```diff
@@ -0,0 +1,245 @@
+# DateTimePicker
+
+<div v-if="hasMounted">
+
+**KDateTimePicker** - A date and time selection tool, displayed inside a popover.
+
+## Examples
+
+### Single date
+
+<KCard>
+  <template v-slot:body>
+    <KDateTimePicker
+      @changed="newVal => currentValue1 = newVal"
+      defaultMessage="Please select a date"
+      calendarMode="date"
+      :isRange="false"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ JSON.stringify(currentValue1) }}</pre></div>
+  </template>
+</KCard>
+
+### Single time
+
+<KCard>
```

Comment:
None of the examples should be inside of a `KCard` component -- they should just appear under the example heading.